README.md

```diff
@@ -1,176 +1,203 @@
-  deployment  Manage your Deployments running on Astronomer
-  dev         Run your Astro project locally
-  help        Help about any command
-  login       Log in to Astronomer
-  logout      Log out of Astronomer
-  version     astro version
-  workspace   Manage Astronomer Workspaces
-
-Flags:
-  -h, --help   help for astro
-
-Use "astro [command] --help" for more information about a command.
-```
+### Core commands
+
+  - `login`       Log in to the Astro CLI
+  - `logout`      Log out of the Astro CLI
+  - `dev init`    Initialize an Astro project in an empty local directory
+  - `dev start`   Build your Astro project into a Docker image and spin up a local Docker container for each Airflow component
+  - `dev stop`    Pause all Docker containers running your local Airflow environment
+  - `dev restart` Stop your Airflow environment, rebuild your Astro project into a Docker image, and restart your Airflow environment with the new Docker image
+  - `deploy`      Deploy code to a Deployment on Astro
+  - `deployment`  Manage your Deployments running on Astronomer
+  - `dev`         Run your Astro project locally
+  - `help`        Help about any Astro CLI command
+  - `version`     Show the running version of the Astro CLI
+  - `workspace`   Manage Astronomer Workspaces
+
+For a list of available Astro CLI commands, see the [Astro CLI command reference](https://docs.astronomer.io/astro/cli/reference).
+
+## Install the Astro CLI
 
-## Install `astro`
+Use these instructions to install, upgrade, or uninstall the Astro CLI.
 
-To install and use the Astro CLI on Mac, you must have:
+### Prerequisites
+
+To install the Astro CLI on Mac, you'll need:
+
+- [Homebrew](https://docs.docker.com/get-docker/)
 - [Docker Desktop](https://docs.docker.com/get-docker/) (v18.09 or higher)
 
-To install and use the Astro CLI on Linux, you must have:
-- [Docker Engine](https://docs.docker.com/engine/install/) (v0.13.1 or higher).
+To install the Astro CLI on Windows, you'll need:
+
+- Windows 10 or later
+- [Docker Desktop for Windows](https://docs.docker.com/desktop/install/windows-install/)
+- [Docker Engine 1.13.1 or later](https://docs.docker.com/engine/install/)
+- [WSL](https://learn.microsoft.com/en-us/windows/wsl/install) enabled on your local machine
+
+To install the Astro CLI on Windows with the Windows Package Manager winget command-line tool, you'll need:
 
-To install and use the Astro CLI on Windows, you must have:
-- Windows 10
-- [Docker Desktop WSL 2 backend](https://docs.docker.com/desktop/windows/wsl/) (v0.13.1 or higher).
+- Windows 10 1709 (build 16299) or later or Windows 11
+- Astro CLI version 1.6 or later
+- The latest version of the Windows [App Installer](https://apps.microsoft.com/store/detail/app-installer/9NBLGGH4NNS1?hl=en-ca&gl=ca)
+- [Docker Desktop for Windows](https://docs.docker.com/desktop/install/windows-install/)
+- [Docker Engine 1.13.1 or later](https://docs.docker.com/engine/install/)
```

Comment:
Same as the above comment

Reply:
@kushalmalani as noted in the previous response, the version information is correct.